pandas_gbq/gbq.py

```diff
@@ -686,6 +750,9 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
         Service account private key in JSON format. Can be file path
         or string contents. This is useful for remote server
         authentication (eg. jupyter iPython notebook on remote host)
+    auth_local_webserver : boolean (default False)
+        Use a local webserver when getting user credentials to handle
```

Comment:
if you can add a doc-link link on how this works would be great (in addition or just to gbq docs is fine as well)